File: supplemental-api-function-app/src/main/java/gov/cdc/ocio/supplementalapi/model/Destination.java
package gov.cdc.ocio.supplementalapi.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Destination {
    @JsonProperty("destination_id")
    public String destinationId;
}

File: supplemental-api-function-app/src/main/kotlin/gov/cdc/ocio/supplementalapi/model/Destination.kt
package gov.cdc.ocio.supplementalapi.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
class Destination {
    @JsonProperty("destination_id")
    var destinationId: String? = null
}